src/app/test-record/test-record.component.spec.ts

```diff
@@ -43,11 +45,16 @@ describe('TestRecordComponent', () => {
     injector = getTestBed();
     component = fixture.componentInstance;
     component.testTypeNumber = 'W01A34247';
-
+    component.testType = testType;
+    component.testRecord = testRecord;
+    component.seatBeltApplicable = ['11'];
+    component.emissionDetailsApplicable = ['22'];
+    component.defectsApplicable = ['33'];
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
```

Comment:
would be nice to add a snapshot and give a bit more meaningful description like we have done.